prover/src/lib.rs

```diff
@@ -219,14 +241,11 @@ pub trait Prover {
         }
     }
 
-    // HELPER METHODS
-    // --------------------------------------------------------------------------------------------
```

Comment:
nit: I would probably keep this separator here.

Reply:
done